text/0000-re-org-docs.md

```diff
@@ -0,0 +1,265 @@
+│       │   ├── build-on-podman.md
+│       │   └── reproducibility.md
+│       └── basic-build
+│           ├── basic-build.md
+│           └── build-phases.md
+├── for-buildpack-authors
+│   ├── how-to-guides
+│   │   ├── migrate <- NOTE: moved from the "reference" section
+│   │   │   ├── buildpack-api-0.4-0.5.md
+│   │   │   ├── buildpack-api-0.5-0.6.md
+│   │   │   ├── buildpack-api-0.6-0.7.md
+│   │   │   ├── buildpack-api-0.7-0.8.md
+│   │   │   ├── buildpack-api-0.8-0.9.md
+│   │   │   └── buildpack-api-0.9-0.10.md
+│   │   ├── package-buildpack
+│   │   │   ├── buildpack-toml.md
+│   │   │   ├── package-buildpack.md
+│   │   │   ├── specify-targets.md
+│   │   │   └── with-clear-env.md <- NOTE: missing content!
+│   │   ├── publish-buildpack
+│   │   │   ├── publish-a-buildpack.md
+│   │   │   └── publishing-with-github-actions.md
+│   │   ├── write-buildpack
+│   │   │   ├── add-labels.md <- NOTE: missing content!
+│   │   │   ├── add-sbom.md
+│   │   │   ├── create-layer.md
+│   │   │   ├── defer-plan-entries.md <- NOTE: missing content!
+│   │   │   ├── re-use-layers.md <- NOTE: missing content!
+│   │   │   ├── specify-env
+│   │   │   │   ├── add-to-posix-paths.md <- NOTE: missing content!
+│   │   │   │   ├── for-build.md <- NOTE: missing content!
+│   │   │   │   ├── for-process.md <- NOTE: missing content!
+│   │   │   │   ├── for-run.md <- NOTE: missing content!
+│   │   │   │   └── with-modifier.md <- NOTE: missing content!
+│   │   │   ├── specify-processes.md
+│   │   │   ├── specify-slices.md <- NOTE: missing content!
+│   │   │   └── verify-targets.md
+│   │   └── write-extension
+│   │       └── advanced-extensions.md
+│   └── tutorials <- NOTE: this would be the ruby buildpack where you have to keep updating the script every time
+│       ├── advanced-buildpack
+│       │   ├── build-plan.md
+│       │   ├── caching.md
+│       │   ├── exec-d.md <- NOTE: missing content!
+│       │   ├── layer-types.md
+│       │   └── processes.md
+│       ├── basic-buildpack
+│       │   ├── build.md
+│       │   ├── building-blocks-cnb.md
+│       │   ├── detection.md
+│       │   └── local-env.md
+│       └── basic-extension
+│           ├── build-dockerfile.md
+│           ├── building-blocks-extension.md
+│           ├── run-dockerfile-extend.md
+│           ├── run-dockerfile-switch.md
+│           ├── setup-local-environment.md
+│           └── why-dockerfiles.md
+├── for-platform-operators
+│   ├── how-to-guides
+│   │   ├── create-a-builder.md
+│   │   ├── create-a-stack.md
+│   │   ├── migrate
+│   │   │   ├── platform-api-0.10-0.11.md
+│   │   │   ├── platform-api-0.11-0.12.md
+│   │   │   ├── platform-api-0.3-0.4.md
+│   │   │   ├── platform-api-0.4-0.5.md
+│   │   │   ├── platform-api-0.5-0.6.md
+│   │   │   ├── platform-api-0.6-0.7.md
+│   │   │   ├── platform-api-0.7-0.8.md
+│   │   │   ├── platform-api-0.8-0.9.md
+│   │   │   └── platform-api-0.9-0.10.md
+│   │   ├── specify-extensions.md
+│   │   └── use-tooling
```

Comment:
Should we be explicit about this section describing integration with other platforms?

```suggestion
│   │   └── CI/CD Integration
```

Reply:
Updated in adb31d5